tests/Unit/RegionTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Adyen\Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use Adyen\Region;
+
+class RegionTest extends TestCase
+{
+    public function testValidRegions()
+    {
+        $expected = [
+            "eu",
+            "us",
+            "au",
+            "in",
+            "apse",
+        ];
+
+        $this->assertEquals(
+            $expected,
+            Region::VALID_REGIONS,
+            "VALID_REGIONS should match the expected regions."
+        );
+    }
+
+    public function testTerminalApiEndpointsMapping()
+    {
+        $expected = [
+            "eu" => "https://terminal-api-live.adyen.com",
+            "us" => "https://terminal-api-live-us.adyen.com",
+            "au" => "https://terminal-api-live-au.adyen.com",
+            "in" => "https://terminal-api-live-in.adyen.com",
```

Comment:
Could you verify whether the`live-in` endpoint is a valid for terminals

Reply:
It's not a valid one, let's remove this one and update the tests https://docs.adyen.com/point-of-sale/design-your-integration/terminal-api/ @DjoykeAbyah

Reply:
Done!